storage/zram.txt

```diff
@@ -0,0 +1,96 @@
+|       <*> Memory allocator for compressed pages                              |
+|                                                                              |
+|   Device Drivers --->                                                        |
+|       <*> Block devices --->                                                 |
+|           <M> Compressed RAM block device support                            |
+|                                                                              |
+|   # To use the lz4 algorithm with zram,                                      |
+|   Cryptographic API --->                                                     |
+|       <*> LZ4 compression algorithm                                          |
+|                                                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+Building compressed RAM block devices as a module is recommended because it
+enables runtime block device creation. If it is built-in to the kernel, pass
+zram.num_devices=X in the kernel commandline (bootloader or built-in).
+
+
+Setup
+________________________________________________________________________________
+
+Compressed block devices can be setup on-the-fly or at boot-time. A quick 
+method using an /etc/rc.d/zram.{boot,pre.shutdown} script is shown below. 
+Because multiple ramdisks are allowed, it might be preferable to create 
+multiple zram devices for better utilization on multicore systems. 
+
+$SIZE below can be given in bytes or suffixed by M,G.
+
++------------------------------------------------------------------------------+
+|   /etc/rc.d/zram.boot                                                        |
++------------------------------------------------------------------------------+
+|   #!/bin/sh -e                                                               |
+|                                                                              |
+|   modprobe zram                                                              |
+|                                                                              |
+|   # create zram devices of $SIZE 2G                                          |
+|   # Mark the devices as swap devices                                         |
+|   # Enable the swap devices                                                  |
+|                                                                              |
+|   for dev in 0 1 2 3; do                                                     |
+|       echo 2G > /sys/block/zram$dev/disksize                                 |
+|       mkswap /dev/zram$dev                                                   |
+|       swapon /dev/zram$dev -p 10                                             |
+|   done                                                                       |
```

Comment:
I know this is a little pedantic but these variables should be quoted. `"/dev/zram$dev"`. Same for the second block of code.

Reply:
Pedantry is my bread and butter.